lms/static/js/dashboard/track_events.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Track interaction with the student dashboard..
+ */
+
+var edx = edx || {};
+
+(function ($) {
+    'use strict';
+
+    edx.dashboard = edx.dashboard || {};
+
+    edx.dashboard.TrackEvents = function() {
+
+         var course_title_link = $(".course-title > a"),
+            course_image_link = $(".cover"),
+            enter_course_link = $(".enter-course"),
+            options_dropdown = $(".wrapper-action-more"),
+            course_learn_verified = $(".message-copy > a"),
```

Comment:
Since it's possible for multiple links to match this selector, you should revert back to using your custom class on this link. Good job not introducing unnecessary classes for these others, though!